src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Repository/Interfaces/IResultsRepository.cs

```diff
@@ -8,6 +8,8 @@ public interface IResultsRepository
     {
         UserPosition GetUserPosition(string userId);
 
-        List<Result> GetTestResultsForWeek(int weekNumber);
+        List<Result> GetTestResultsForWeek(int weekNumber, int resultsCountToGet, int paginationIndex);
```

Comment:
I suggest a variable with a name _pageSize_, not _resultsCountToGet_

Reply:
It's the repository so I don't think that view specific term "page" is suitable here.